File: algorithm/src/main/java/baekjoon/p02480/Main.java
package baekjoon.p02480;

import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int a = sc.nextInt();
        int b = sc.nextInt();
        int c = sc.nextInt();

        System.out.println(solve(a, b, c));
    }

    private static int solve(int a, int b, int c) {
        int[] arr = new int[7];
        arr[a]++;
        arr[b]++;
        arr[c]++;

        int max = 1;
        for (int i = 0; i < arr.length; i++) {
            if (arr[i] == 3) {
                return (10000 + i * 1000);
            }

            if (arr[i] == 2) {
                return (1000 + i * 100);
            }

            if (arr[i] == 1) {
                max = i;
            }
        }

        return (max * 100);
    }

}

File: algorithm/src/main/java/baekjoon/p02480/Main.kt
package baekjoon.p02480

fun main(args: Array<String>) {
    val input = readLine()!!.split(" ")
    val a = input[0].toInt()
    val b = input[1].toInt()
    val c = input[2].toInt()

    println(solve(a, b, c))
}

private fun solve(a: Int, b: Int, c: Int): Int {
    val arr = arrayOfNulls<Int>(7)
    arr.fill(0)

    arr[a] = arr[a]!!.inc()
    arr[b] = arr[b]!!.inc()
    arr[c] = arr[c]!!.inc()

    var max = 1
    for (i in 1..6) {
        if (arr[i] == 3) {
            return (10000 + i * 1000)
        }

        if (arr[i] == 2) {
            return (1000 + i * 100)
        }

        if (arr[i] == 1) {
            max = i
        }
    }

    return (max * 100)
}